src/chrome/content/rules/dei.gr.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="dei.gr (partial)">
+	<target host="www.dei.gr" />
```

Comment:
```
elearning.dei.gr
sapwasrmprd.dei.gr

Reply:
@cschanaj not sure if you also noticed [this](https://github.com/EFForg/https-everywhere/pull/14352#discussion_r162782630) comment.